_config.yml

```diff
@@ -277,6 +277,9 @@ mathjax:
 # Gentie productKey
 #gentie_productKey:
 
+# youyan uid
```

Comment:
I write a blog to explain how to implement ,
https://jinfang134.github.io/posts/5f6c0475/
sorry for it is writen in chinese.